docs/api/products.rst

```diff
@@ -1,33 +1,168 @@
+
++----------------+----------------------------------------------------------------------------+
+| Field          | Description                                                                |
++================+============================================================================+
+| id             | Id of product                                                              |
++----------------+----------------------------------------------------------------------------+
+| code           | Unique product identifier                                                  |
++----------------+----------------------------------------------------------------------------+
+| average_rating | Average from ratings given by customer                                     |
++----------------+----------------------------------------------------------------------------+
+| channels       | Collection of channels to which product was assigned                       |
++----------------+----------------------------------------------------------------------------+
+| translations   | Collection of translations (each contains slug and name in given language) |
++----------------+----------------------------------------------------------------------------+
+
+If you request for a more detailed data, you will receive an object with following fields:
+
++----------------+----------------------------------------------------------------------------+
+| Field          | Description                                                                |
++================+============================================================================+
+| id             | Id of product                                                              |
++----------------+----------------------------------------------------------------------------+
+| code           | Unique product identifier                                                  |
++----------------+----------------------------------------------------------------------------+
+| average_rating | Average from ratings given by customer                                     |
++----------------+----------------------------------------------------------------------------+
+| channels       | Collection of channels to which product was assigned                       |
++----------------+----------------------------------------------------------------------------+
+| translations   | Collection of translations (each contains slug and name in given language) |
++----------------+----------------------------------------------------------------------------+
+| attributes     | Collection of attributes connected with product (for example material)     |
++----------------+----------------------------------------------------------------------------+
+| variants       | Collection of variants connected with product                              |
```

Comment:
By "default" you mean default serialization group or generally the serialization in Sylius?